adobe/aio-lib-runtime
# fix: remove the webpack config module from the require cache

This is for the situation where if you have a file watcher process, and you need to rebuild an action due to a webpack-config.js change.

Closes #35

## How Has This Been Tested?

1. Add a `actions/webpack-config.js` file, exporting  a CommonJS module as a webpack config
2. Run ` DEBUG=@adobe/aio-lib-runtime:action-builder:debug aio app run`
3. See the log of the web pack config being loaded
4. Wait until the 2 urls of your deployed application is logged
5. Edit `actions/webpack-config.js` with a new entry/delete an entry
6. See the log of the web pack config being loaded, it should be different

`npm test`

## Types of changes




## Checklist:
